Core/Contributions/Camp Smalltalk/ANSI/Tests/AnsiDB.st

```diff
@@ -2459,7 +2459,7 @@ ProtocolSpec newMessagePattern: 'year: hour day: dayOfYear hour: minute minute:
 	definedIn: 'DateAndTime factory'
 	definition: 'Answer the least <DateAndTime> which is the second second of the minute minute of the hour hour of the day dayOfYear of the year year of the astronomical Gregorian calendar in the local time specified by the implementation. The second must be a <number> greater than or equal to 0 and strictly less than 60. The minute must be an <integer> between 0 and 59 inclusive. The hour must be an <integer> between 0 and 23 inclusive. The day must be an <integer> between 1 and 366 inclusive. An implementation may not impose any limits on the year other than those imposed on <integer> constants.
 
-It is possible that the time specified does not exist in the local time specified by the implementation. If there is a time change such that the local time is set forward and the time specified is in the interregnum, then that time does not exist in the local time. For example if at 02:00 in California on April 26, 1997 there is a time change that sets local time forward one hour, then the local time 02:30  in California does not exist. Conversely if there is a time change that sets the locale time back there are times which are ambiguous. For example if instead of setting the local time forward from 02:00 to 03:00 it is set back to 01:00 the the local time 01:30 in California is ambiguious. The result is the least <DateAndTime> that conforms to the given parameters.
+It is possible that the time specified does not exist in the local time specified by the implementation. If there is a time change such that the local time is set forward and the time specified is in the interregnum, then that time does not exist in the local time. For example if at 02:00 in California on April 26, 1997 there is a time change that sets local time forward one hour, then the local time 02:30  in California does not exist. Conversely if there is a time change that sets the locale time back there are times which are ambiguous. For example if instead of setting the local time forward from 02:00 to 03:00 it is set back to 01:00 the local time 01:30 in California is ambiguious. The result is the least <DateAndTime> that conforms to the given parameters.
```

Comment:
In some cases 'the the' is not a doubling up, but rather a mistyping of 'then the'. That is the case here. It either needs the 'then', or a comma before 'the'. I'm not sure why this paragraph is repeated so much, but disregarding that the same would apply to the duplicates too.